widget/entry.go

```diff
@@ -403,6 +415,16 @@ func (e *Entry) SelectedText() string {
 	return string(e.textProvider().buffer[start:stop])
 }
 
+// SetAcceptTabs sets if Entry will accept tabs for text entry.
+// This is overrides the default behaviour of Multiline Entries accepting tabs
+func (e *Entry) SetAcceptTabs(acceptTabs bool) {
+	if acceptTabs {
+		e.acceptTabs = 1
```

Comment:
You can add a `return` here and remove the else statement.

Reply:
Done